vault/request_handling_util.go

```diff
@@ -68,12 +68,8 @@ func possiblyForwardAliasCreation(ctx context.Context, c *Core, inErr error, aut
 
 var errCreateEntityUnimplemented = "create entity unimplemented in the server"
 
-func possiblyForwardEntityCreation(ctx context.Context, c *Core, inErr error, auth *logical.Auth, entity *identity.Entity) (*identity.Entity, error) {
-	return entity, inErr
-}
-
-func updateLocalAlias(ctx context.Context, c *Core, auth *logical.Auth, entity *identity.Entity) error {
-	return nil
+func registerLocalAlias(_ context.Context, _ *Core, _ *logical.Alias, inEntity *identity.Entity, inError error) (*identity.Entity, error) {
```

Comment:
Why inError?

Reply:
And for that matter, why inEntity?

Reply:
So that we don't return 'nil, nil'. This is only for the case this is called from the CE version.

Reply:
How about instead, remove the extra params and make the CE registerLocalAlias simply return ErrReadOnly unconditionally?  That's the only possible error anyway.  

In a way it's a moot point, because there are no perf standbys on CE, so we shouldn't be getting ErrReadOnly due to attempts to write entities on non-primary-active-nodes.  I just think it's weird done like this, and undesirable from a readability perspective on the ent side (as a reader, not stepping into the code: "the entity returned by CreateOrFetchEntity is apparently an input to registering local aliases".)

Reply:
Now that I look more in that part of the code, I see we're doing something similar for possiblyForwardAliasCreation, but there we're actually moving all the logic around checking for ErrReadonly into the ent method.  I'd be ok with that approach too.  Alternatively, you could actually move the entire body of the `if err != nil` check after calling CreateOrFetchEntity into a single func that handles ErrReadOnly in both the local and non-local cases.